cbirrt2/cbirrtparameters.h

```diff
@@ -90,12 +90,17 @@ class CBirrtParameters : public PlannerBase::PlannerParameters
         if( !PlannerParameters::serialize(O) )
             return false;
 
+        std::streamsize old_precision = O.precision();
+        O.precision(17);

```

Comment:
I suggest using `std::numeric_limits<OpenRAVE::dReal>:: max_digits10` (if you're okay with C++11) or `std::numeric_limits<OpenRAVE::dReal>:: digits10 + 1` (for C++98) instead of hardcoding `17`.